app/views/AdminView/components/VariantText/columnDefs.ts

```diff
@@ -0,0 +1,49 @@
+import { formatDate } from '@/utils/date';
+import { ColDef } from '@ag-grid-community/core';
+
+const columnDefs: ColDef[] = [
+  {
+    headerName: 'Template Name',
+    field: 'template.name',
+  },
+  {
+    headerName: 'Project',
+    cellRenderer: ({ data }) => data.project?.name || '(default appendix text)',
+  },
+  {
+    headerName: 'Created',
+    valueGetter: ({ data }) => formatDate(data.createdAt),
+    minWidth: 90,
+  },
+  {
+    headerName: 'Updated',
+    valueGetter: ({ data }) => formatDate(data.updatedAt),
+    minWidth: 90,
+  },
+  {
+    headerName: 'Variant Name',
+    valueGetter: ({ data }) => data.variantName,
+    minWidth: 90,
+  },
+  {
+    headerName: 'Cancer Type',
+    valueGetter: ({ data }) => data.cancerType,
+    minWidth: 90,
+  },
+  {
+    headerName: 'Text',
+    cellRenderer: ({ data }) => (data.text ? `${data.text.substring(0, 300)}....` || '' : ''),
```

Comment:
I'm not sure about this, due to having the text `....` always being there if string length less <300, I can understand where the motivation comes from, but I don't think it is working as intended

the `....` is there for when the text is overflown and blocked off, but the way the table is rendered makes it so that the whole text cell is rendered and brought to size, making this .... an artefact

had the cell been a 1 row height instead, the user can click this cell and expand the text, but in this case the text is already expanded so I think we can leave the .... out unless you decide to make this fancy and expand out